music-player/song.js

```diff
@@ -1 +1,17 @@
 // Your code here
+
+function Song() {
+
+}
+
+function Album() {
+
+}
+
+function Artist() {
+
+}
+
+function Playlist(name) {
+  this.name = name;
+}
```

Comment:
In the future try patching the files that get added so that you dont have files like these that dont really have any functionality to them